### PR TITLE
Fwmt 2020 2022 2023 add new views

### PR DESCRIPTION
**Pull request by SteveScorfield**

### CODE REVIEW

File: app/views/logistics_view.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+
+def get_employee_tabs(employee_information, current_job_role, device_information):
+
+    employee_devices, device_number = device_details(device_information)
+
+    employee_name = map_employee_name(employee_information)
+
+    if employee_information['preferredName'] == '':
+        preferred_name = employee_information['preferredName']
```
Comment (sdiaz1000) [inline app/views/logistics_view.py]: same as above

File: app/views/rmt_view.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+def get_employee_tabs(employee_information, current_job_role, device_information):
+
+    employee_devices, device_number = device_details(device_information)
+
+    line_manager = format_line_manager(current_job_role)
+
+    employee_name = map_employee_name(employee_information)
+
+    if employee_information['preferredName'] == '':
+        preferred_name = employee_information['preferredName']
```
Comment (sdiaz1000) [inline app/views/rmt_view.py]: as above

**Review by sdiaz1000 — CHANGES_REQUESTED** [review]

File: app/views/hr_view.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+
+def get_employee_tabs(employee_information, current_job_role, device_information):
+
+    employee_devices, device_number = device_details(device_information)
+
+    employee_name = map_employee_name(employee_information)
+
+    if employee_information['preferredName'] == '':
+        preferred_name = employee_information['preferredName']
```
[the remark this answers is not in the document]
Comment (SteveScorfield) [reply app/views/hr_view.py]: You are correct. Have altered this.

File: app/views/logistics_view.py (hunk 1 above)
Comment (SteveScorfield) [reply app/views/logistics_view.py]: As above

File: app/views/rmt_view.py (hunk 2 above)
Comment (SteveScorfield) [reply app/views/rmt_view.py]: As above

File: app/views/hq_fo_ccs_view.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+def get_employee_tabs(employee_information, current_job_role, device_information):
+
+    employee_devices, device_number = device_details(device_information)
+
+    line_manager = format_line_manager(current_job_role)
+
+    employee_name = map_employee_name(employee_information)
+
+    if employee_information['preferredName'] == '':
```
[the remark this answers is not in the document]
Comment (SteveScorfield) [reply app/views/hq_fo_ccs_view.py]: As. below

**Review by sdiaz1000 — APPROVED** [review]

**Pull request merged**